doc/lua_api.txt

```diff
@@ -6113,20 +6245,26 @@ Used by `minetest.register_decoration`.
                  ...
             },
         },
-    --  ^ See 'Schematic specifier' for details.
+        -- Could instead be a table containing two mandatory fields, size and
```

Comment:
More like: `Alternative schematic specification by supplying a table. The fields size and data are mandatory whereas yslice_prob is optional`

Reply:
+1